subsystem/kubeapi_test.go

```diff
@@ -134,6 +156,30 @@ func waitForClusterDeploymentCRDState(
 	Expect(successInARaw).Should(Equal(minSuccessesInRow))
 }
 
+func waitForInstallEnvCRDState(
+	ctx context.Context, client k8sclient.Client, key types.NamespacedName, state string, timeout int) {
+
+	installEnv := &v1alpha1.InstallEnv{}
+	successInARaw := 0
+	start := time.Now()
+
+	for time.Duration(timeout)*time.Second > time.Since(start) {
+		installEnv = getInstallEnvCRD(ctx, client, key)
+		if len(installEnv.Status.Conditions) > 0 && installEnv.Status.Conditions[0].Message == state {
```

Comment:
same as above i'm not sure that the order here is guarantied, probably need to search for specific condition.

Reply:
Done.